static/app/views/alerts/rules/metric/thresholdTypeForm.tsx

```diff
@@ -78,7 +81,10 @@ function ThresholdTypeForm({
     ],
   ];
 
-  if (hasAnomalyDetection) {
+  if (
+    hasAnomalyDetection &&
+    !(alertType === 'apdex' || alertType === 'llm_tokens' || alertType === 'llm_cost')
```

Comment:
Can we do the inverse? So only show if it the alert types are the ones we know work (everything except these)?

Reply:
Sure, I was just concerned that it would be messy. Do you think it's okay?